vendor/index.yaml

```diff
@@ -1552,6 +1552,10 @@ vendors:
     name: Exedra General Trading and Contracting
     vendorID: 841
 
+  - id: sensecap
+    name: Seeed Technology Co.,Ltd.
+    vendorID: 888
```

Comment:
We keep https://lora-alliance.org/wp-content/uploads/2020/11/LoRa_Alliance_Vendor_ID_for_QR_Code.pdf as the authoritive list. Are you a LoRa Alliance member with vendor ID 888? If so, please send us a confirmation via email. Otherwise, please remove the `vendorID` field here.